### PR TITLE
add share facebook function and update user's avatar

### PR DESCRIPTION
**Pull request by thangdangminh**

### CODE REVIEW

**Comment by thangdangminh** [comment]
ready

**Comment by tranducquoc** [comment]
commented
conflict

**Comment by thangdangminh** [comment]
ready

File: resources/views/shop/show.blade.php [hunk 1]
```diff
@@ -1,6 +1,14 @@
 @extends('layouts.app')
 @section('title', $product->name)
 
+@section('meta-fb-share')
+    <meta property="og:url"           content="{{ URL::current() }}" />
```
Comment (tranducquoc) [inline resources/views/shop/show.blade.php]: cách 1 space thôi chứ

**Comment by thangdangminh** [comment]
ready

**Pull request merged**